### PR TITLE
XD68 - Adjust the VIA keymap to more closely match vendor

### PR DESCRIPTION
**Pull request by drowe**
## Description

The vendor for this board displays many typical layouts
and supported keymaps.  The typical right side layout
for this size of board would have ALT, FN, CTRL, rather
than an 'App' key of unknown functionality.

This adjusts the default for VIA enabled boards, such that
someone getting started with this board and VIA would have
a known base layout.

Possible layouts shown here - https://kprepublic.com/products/xiudi-xd68-pcb-65-custom-mechanical-keyboard-support-tkg-tools-underglow-rgb-pcb-programmed-kle-lots-of-layouts

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by zvecr — APPROVED** [review]

**Comment by fauxpark** [comment]
Just as a note, the "App" key is actually the Windows context menu key. I like to have it combined with the FN key as a layer tap, but I'm not sure VIA natively supports those just yet.

**Review by fauxpark — APPROVED** [review]

**Comment by drowe** [comment]
> Just as a note, the "App" key is actually the Windows context menu key. I like to have it combined with the FN key as a layer tap, but I'm not sure VIA natively supports those just yet.

There is a 'Menu' key, which is mapped to `KC_APP` - which is what it was before, so that should be mappable using an Fn key - my goal here was to more closely match the vendor keymaps listed.

**Review by drashna — APPROVED** [review]

File: keyboards/xd68/rules.mk [hunk 1]
```diff
@@ -32,4 +32,4 @@ FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
 UNICODE_ENABLE = no         # Unicode
 
-LAYOUTS = 65_ansi 65_iso
\ No newline at end of file
+LAYOUTS = 65_ansi 65_iso all
```
Comment (fauxpark) [inline keyboards/xd68/rules.mk:35]:
```suggestion
LAYOUTS = 65_ansi 65_iso
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/xd68/rules.mk (hunk 1 above)
Comment (drowe) [reply keyboards/xd68/rules.mk]: Thanks for that catch - I just read up on the docs for that var, and this shouldn't have been in there.

**Review by fauxpark — APPROVED** [review]

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**